.github/workflows/examples.yml

```
@@ -0,0 +1,89 @@
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '22'
+          cache: 'npm'
+
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)